# Copyright (c) 2022-2023 Antmicro <www.antmicro.com>
#
# SPDX-License-Identifier: Apache-2.0

int32 index
---
camera_node/CameraPropertyDetails details
